tests/InferTypesTest.elm

```diff
@@ -30,86 +31,46 @@ typeInference =
         (List.map runSection
             [ ( "list"
               , [ ( "empty list"
-                  , Canonical.List []
-                  , Ok ( Typed.List [], Type.List (Type.Var 1) )
+                  , located { end = { col = 3, row = 1 }, start = { col = 1, row = 1 } } (Canonical.List [])
+                  , Ok (located { end = { col = 3, row = 1 }, start = { col = 1, row = 1 } } ( Typed.List [], List (Var 1) ))
```

Comment:
Can we try modifying `runTest` to `Typed.getType`, so that we wouldn't have to show all that location info in the tests?

We would then not really test the expression themselves (eg. that `Canonical.List [a,b,c]` changes into `Typed.List [a,b,c]`), but I think the main thing is to test the type inference, thus the types... And we'd get rid of the location info! :tada:

```elm
input
  |> Stage.InferTypes.inferExpr
  |> Typed.getType
  |> Expect.equal output
```

Reply:
Located is within the Expr(_) type, so unless we unwrap recursively I don't think we can get rid of those.

Reply:
if we do `Typed.getType`, we get rid of all Exprs, so no LocatedExprs either anymore